### PR TITLE
Add support for css width and height

### PR DESCRIPTION
**Pull request by Jbithell**
Changes proposed in this Pull Request:
- Move to CSS width and height attribute - in the world of bootstrap etc. it would be very useful to be able to specify the width and height of the images - instead of using the depreciated width= and height= tag attributes in a fixed way

@adrianengine

### CODE REVIEW

**Comment by Jbithell** [comment]
Now tested on a dev site and working

**Comment by adrianengine** [comment]
Hello @Jbithell 

Can you provide some use cases where the `width` and `height` attributes on the image are a issue?

Those attributes are there since according to Page Speed Insights:

> A width and height should be specified for all images in order to speed up 
page display

You still can change the images `width` and `height` via your CSS and those attributes will not affect anything.

Thank You!

P.D I will review your PR anyway and make some comments

File: spectragram.js [hunk 1]
```diff
@@ -121,6 +121,8 @@ if ( typeof Object.create !== "function" ) {
                 imageWidth,
                 max,
                 setSize,
+		WidthSetting,
```
Comment (adrianengine) [inline spectragram.js]: Please name this variables with `camelCase` format.

```
heightSetting,
widthSetting
```

File: spectragram.js [hunk 2]
```diff
@@ -121,6 +121,8 @@ if ( typeof Object.create !== "function" ) {
...
+		HeightSetting,
```
Comment (adrianengine) [inline spectragram.js]: Please order variables alphabetically

File: spectragram.js [hunk 3]
```diff
@@ -156,10 +160,7 @@ if ( typeof Object.create !== "function" ) {
 
 					$image = $( "<img>", {
 						alt: imageCaption,
-						attr: {
-							height: imageHeight,
-							width: imageWidth
-						},
+						style: "width: " + (WidthSetting == null ? imageWidth + "px" : WidthSetting) + "; height: " + (HeightSetting == null ? imageHeight + "px": HeightSetting),
```
Comment (adrianengine) [inline spectragram.js]: This line is above 120 characters, consider splitting it to make it more readable

File: spectragram.js [hunk 4]
```diff
@@ -211,7 +212,9 @@ if ( typeof Object.create !== "function" ) {
 		max: 10,
 		query: "instagram",
 		size: "medium",
-		wrapEachWith: "<li></li>"
+		wrapEachWith: "<li></li>",
+	    	width: null,
```
Comment (adrianengine) [inline spectragram.js]: Please order variables alphabetically

File: spectragram.js (hunk 3 above)
Comment (adrianengine) [inline spectragram.js]: Please use `===` to compare with `null`

**Comment by Jbithell** [comment]
@adrianengine thanks the review - looks much better than my version!

My use case was where the content was being displayed in a bootstrap `<div class="col-sm-3 col-md-2 col-lg-2">` etc. and therefore the image being 100% width was quite important - but the nature of the theme, and its overuse of CSS `!important` _*grumbles internally*_ meant overriding the styling was quite tricky. 

I agree width and height are also important for page speed - so I'm quite happy to keep my fork seperated if you'd prefer

**Comment by adrianengine** [comment]
Thanks @Jbithell 
Since is not a breaking issue, i will leave it as it is.

See ya!

**Pull request closed without merging**